core/src/main/java/org/jdbi/v3/core/Jdbi.java

```diff
@@ -363,6 +365,15 @@ public Handle open() {
             LOG.trace("Jdbi [{}] obtain handle [{}] in {}ms", this, h, MILLISECONDS.convert(stop - start, NANOSECONDS));
             return h;
         } catch (SQLException e) {
```

Comment:
This will not catch exception thrown from Jdbi's PostgresPlugin. This one throws `LargeObjectException`

Reply:
Good point, updated.

Reply:
@kristoffSC please take another look and see if it looks good now :)